src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Capture Donor Intent in GS for End User</shortDescription>
+        <value>When you raise funds for a specific purpose, you want to be sure you are honoring the donor intent. Ensure donations are allocated correctly with General Allocation Units.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Capture Donor Intent in GS for End User</shortDescription>
+        <value>Learn more about Donation Allocations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Capture Donor Intent in GS for End User</shortDescription>
+        <value>Create Allocations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCaptureDonorIntentTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Capture Donor Intent in GS for End User</shortDescription>
+        <value>Capture Donor Intent with Donation Allocations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Monthly giving can provide a predictable source of nonprofit revenue, and sustaining donors are often a nonprofit&apos;s most loyal and committed supporters. Track monthly giving by creating a Recurring Donation.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Collect Gifts in GS for End User</shortDescription>
+        <value>Learn more about Recurring Donations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemCollectGiftsPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Collect Gifts in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Primary button label of Collect Gifts checklist item in Get Started for End Users.